app/scss/styles.scss

```diff
@@ -29,50 +54,77 @@ html {
 
       .user_list {
         padding: 20px;
-
-        .btn_height {
-          padding: 0 1px 0 1px;
-        }
       }
+    }
 
-      .score_list {
-        padding: 20px;
-      }
+    .score_list {
+      padding: 20px;
+    }
 
-      img {
-        width: 100%;
-      }
+    img {
+      width: 100%;
+    }
 
-      .logo {
-        width: 300px;
-        height: 100px;
-        padding: 25px;
-        float: right;
-        margin: 10px;
-      }
+    .logo {
+      width: 300px;
+      height: 100px;
+      padding: 30px;
+      float: right;
+      margin: 10px;
+    }
 
-      p {
-        display: inline;
-        margin: 10px;
-      }
+    p {
+      display: inline;
+      margin: 10px;
+    }
 
-      .primary-button {
-        background-color: $primary-button;
-        color: $main-background-color;
-        transition: .5s;
-        border-radius: 0;
-        padding: 6px 12px;
+    .custom-select {
+      background-color: white;
+      font-size: 1.5rem;
+    }
 
-        &:hover {
-          background-color: $primary-button-hover;
-        }
-      }
+    .primary-button {
+      background-color: $pink-primary-button;
+      color: $main-background-color;
+      transition: .25s;
+      border-radius: 4px;
+      padding: 6px 12px;
+      @include buttonsMetaCapsFontStyles;
 
-      .search-heading {
-        font-size: 1.5rem;
-        margin: 0;
+      &:hover {
+        background-color: $primary-button-hover;
+        color: $main-background-color;
       }
+    }
+
+    .edit-user-button:hover,
+    .more-info-button:hover,
+    .view-results-button:hover,
+    .download-user-results-button:hover {
+      background-color: $energyGreen;
+      color: $font-color;
+      transition-duration: .25s;
+    }
+
+    .delete-user-button:hover {
+      background-color: $energyGreen;
+      color: $font-color;
+      transition-duration: .25s;
+    }
 
+    .search-heading {
+      font-size: 1.5rem;
+      margin: 0;
     }
   }
+
+  .editUserData {
+    font-size: 1.5rem;
+  }
+
+}
+
+.no-flex {
+  display: block;
+  //margin-right: 40px;
 }
```

Comment:
uhm what is this? someone explain to me why this is needed, also commented out code should be deleted